src/main/java/com/maxdemarzi/shortest/Validators.java

```diff
@@ -22,6 +22,10 @@ public static HashMap getValidQueryInput(String body) throws IOException {
         if (!input.containsKey("center_email")) {
             throw Exceptions.missingCenterEmailParameter;
         }
+        // Make sure it has a bibliography_entries parameter
+        if (!input.containsKey("bibliography_entries")) {
```

Comment:
i guess this means we expect downtime when we deploy this, until modify the Sidekiq job and email service to send along bibliography entry IDs? i guess that's ok if we quickly modify the email service to send an empty array